src/dr_llm/project/docker_psql.py

```
@@ -60,46 +61,66 @@ def _call_docker_psql_admin(
 ) -> None:
-    args = (
-        "exec",
-        "-i",
-        container_name,
-        "psql",
-        "-U",
-        db_user,
-        db_name,
-    )
     try:
         process = subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)